cmd/syncthing/main.go

```diff
@@ -451,7 +451,7 @@ func main() {
 func openGUI() {
 	cfg, _ := loadConfig()
 	if cfg.GUI().Enabled {
-		openURL(cfg.GUI().URL())
+		openURL(cfg.GUI().URLs()[0])
```

Comment:
This won't work if the first address will be UNIX socket for example. Not saying you have to fix this now, but keep this in mind.